fiducialMarker/fiducialMarker.py

```diff
@@ -43,6 +43,10 @@ class AeroCubeFace(Enum):
         LEFT = 5
 
     @staticmethod
+    def _valid_aerocube_ID(ID):
+        return ID >= 0 and ID*AeroCubeMarker._NUM_AEROCUBE_SIDES + AeroCubeMarker._NUM_AEROCUBE_SIDES <= AeroCubeMarker.get_dictionary_size()

```

Comment:
Broke it up into multiple lines -- may want to make a helper function for base_marker and end_marker in the future though.